migration-engine/connectors/sql-migration-connector/src/sql_database_step_applier.rs

```diff
@@ -100,6 +100,9 @@ impl DatabaseMigrationStepApplier for SqlMigrationConnector {
     #[tracing::instrument(skip(self, script))]
     async fn apply_script(&self, migration_name: &str, script: &str) -> ConnectorResult<()> {
         tracing::info!(migrate_action = "log", "Applying migration `{}`", migration_name);
+        self.host
+            .print(&format!("Applying migration `{}`", migration_name))
```

Comment:
Are we doing a double output here? Line 102 and 103-105...

Reply:
indeed, I forgot to remove 102, good catch!